joomla/joomla-cms
# [5.0] Update deleted files and folders and renamed files for 5.0 and remove 4.x update SQL scripts

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the lists of deleted files and folders and renamed files in script.php for Joomla 5 and removes the 4.x update SQL scripts.

Both is made under the assumption that updating to Joomla 5 will only be supported from the latest 4.4 release, like it is for updating to Joomla 4 from the latest 3.10 release.

In detail following changes are made:
- Clear the lists of deleted files and folders and renamed files in script.php by old entries for updating from 3 to 4.
- Add deleted files and folders from #38406 .
- Add deleted file from #38405 .
- Add deleted files and folders from #39134 .
- Add deleted files and folders and renamed files from dependency updates.
- Remove the 4.x update SQL scripts (106 files in total, that's why this PR shows 109 changed files on GitHub) and add them to the deleted files list.
- Add an initial 5.0.0 update SQL script for each database type deleting the `#__utf8_conversion` table as suggested here: https://github.com/joomla/joomla-cms/pull/38406#issuecomment-1209286799 . The table should have been deleted long ago, but to be safe we delete it again, if it still exists.

### Testing Instructions

Code review.

Or if you want to make a real test, update the latest 4.4-dev nightly build to the last 5.0-dev nightly build to get the actual result, and update the latest 4.4-dev nightly build to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

- Deleted files and folders from PRs #38406 , #38405 , #39134 and diverse dependencies are still present after updating from the latest 4.4-dev nightly build.
- The 4.x update SQL scripts are still present.
- Files "ED256.php" and "ED512.php" in folder "/libraries/vendor/web-auth/cose-lib/src/Algorithm/Signature/EdDSA" have not been renamed to "Ed256.php" and "Ed512.php"

### Expected result AFTER applying this Pull Request

- Deleted files and folders from PRs #38406 , #38405 , #39134 and diverse dependencies have been deleted after updating from the latest 4.4-dev nightly build.
- The 4.x update SQL scripts have been deleted.
- Files "ED256.php" and "ED512.php" in folder "/libraries/vendor/web-auth/cose-lib/src/Algorithm/Signature/EdDSA" have been renamed to "Ed256.php" and "Ed512.php"

### Link to documentations
Please select:
